src/styles/main.css

```diff
@@ -0,0 +1,726 @@
+    flex-direction: row;
+    padding: 20px 40px;
+    margin: 0;
+    width: 100%;
+    background-color: rgb(255, 255, 255);
+  }
+
+  .navbar ul li {
+    display: inline;
+    list-style: none;
+    font-family: 'lato', Arial, Helvetica, sans-serif;
+    font-weight: bolder;
+    font-style: normal;
+    font-size: 15px;
+    line-height: 20px;
+    color: rgb(52, 69, 99);
+    margin-right: 30px;
+  }
+
+  .navbar ul li a {
+    text-decoration: none;
+    color: var(--gray-dark);
+  }
+
+  .navbar ul li a:hover {
+    color: var(--orange);
+  }
+
+  .item--button {
+    color: var(--orange) !important;
+    font-weight: bold;
+    border: solid 3px var(--orange);
+    padding: 10px 30px;
+    transition: all 0.3s;
+  }
+
+  .item--button:hover {
+    color: var(--white) !important;
+    background-color: var(--orange);
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .header--content {
+    width: 100%;
+  }
+
+  .header--content--title {
+    padding-left: 70px;
+    margin: 0;
+  }
+
+  .header--content--text {
+    width: 50%;
+    align-self: flex-start;
+    margin-left: 70px;
+  }
+
+  .header--content-location {
+    width: 50%;
+    align-self: flex-start;
+    margin-left: 70px;
+  }
+
+  .header--content--main {
+    height: 200px;
+  }
+
+  .programs {
+    flex-direction: column;
+    padding: 10px;
+  }
+
+  .programCards {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .programCard {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    text-align: center;
+    align-items: center;
+    background-color: var(--gray-light);
+    opacity: 0.7;
+    height: 250px;
+    gap: 15px;
+    margin-bottom: 10px;
+    width: 200px;
+    transition: transform 0.3s;
+  }
+
+  .programCard:hover {
+    outline: solid 5px var(--white);
+    transform: translateY(-8px);
+  }
+
+  .programCard i {
+    color: var(--orange);
+    font-size: 40px;
+    font-weight: bold;
+    margin-top: 30px;
+  }
+
+  .programCard--title {
+    font-family: 'lato', Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    font-weight: bold;
+    color: var(--orange);
+  }
+
+  .programCard--text {
+    flex: 1;
+    width: 100%;
+    padding: 10px;
+  }
+
+  .speakersGrid {
+    display: grid;
+    grid-template-columns: 350px 350px;
+    grid-template-rows: 130px 130px 130px;
+    justify-content: center;
+    align-items: center;
+    padding: 20px 20px;
+    gap: 30px;
+  }
+
+  .speakerCard {
+    height: 130px;
+    width: 350px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 20px;
+    gap: 10px;
+  }
+
+  .speakers--button {
+    display: none;
+  }
+
+  #menuAbout {
+    display: none;
+  }
+
+  .pastSummits {
+    flex-direction: column;
+  }
+
+  .pastSummits--seconde {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+  }
+}
```

Comment:
- [Optional] Based on the requirements of this project, pages use the exact same colors, for text, backgrounds, borders, etc, as the original design. You haven't made use of these basic color swatches: #272A31, #EC5242, #D3D3D3